lascli/parser/transitions.py

```diff
@@ -0,0 +1,97 @@
+from las import Client
+import json
+import pathlib
+
+
+def create_transition(las_client: Client, name, transition_type,
```

Comment:
Should we accept non path arguments for params as well?

Reply:
Good question, I did it like this because it was the simplest and fastest way to do it during developing the API. 

That being said, I fear that the params are a bit to diverse to allow for a smooth way of specifying them explicitly:

Here are a couple of examples:

`{
  "assets": {
    "jsRemoteComponent": "las:asset:<hex>",
    "companies": "las:asset:<hex>",
    "suppliers": "las:asset:<hex>",
    "data": "las:asset:<hex>"
  }
}
`  

`{
  "imageUrl": "<image-url>",
  "secretId": "las:secret:<hex>",
  "memory": 1024,
  "cpu": 256 
}
`